cirq/protocols/json_test_data/Calibration.json

```diff
@@ -0,0 +1,4 @@
+{
+  "cirq_type": "Calibration",
+  "metrics": "{\n  \"metrics\": [\n    {\n      \"name\": \"xeb\",\n      \"targets\": [\n        \"0_0\",\n        \"0_1\"\n      ],\n      \"values\": [\n        {\n          \"doubleVal\": 0.9999\n        }\n      ]\n    },\n    {\n      \"name\": \"xeb\",\n      \"targets\": [\n        \"0_0\",\n        \"1_0\"\n      ],\n      \"values\": [\n        {\n          \"doubleVal\": 0.9998\n        }\n      ]\n    },\n    {\n      \"name\": \"t1\",\n      \"targets\": [\n        \"0_0\"\n      ],\n      \"values\": [\n        {\n          \"int64Val\": \"321\"\n        }\n      ]\n    },\n    {\n      \"name\": \"t1\",\n      \"targets\": [\n        \"0_1\"\n      ],\n      \"values\": [\n        {\n          \"int64Val\": \"911\"\n        }\n      ]\n    },\n    {\n      \"name\": \"t1\",\n      \"targets\": [\n        \"1_0\"\n      ],\n      \"values\": [\n        {\n          \"int64Val\": \"505\"\n        }\n      ]\n    },\n    {\n      \"name\": \"globalMetric\",\n      \"values\": [\n        {\n          \"strVal\": \"abcd\"\n        }\n      ]\n    }\n  ]\n}"
```

Comment:
What is the format of the string value under the "metrics" key? It looks like JSON... If so, it's a little ugly to embed JSON as string in another JSON. Could we use a proper JSON dictionary as a value for the "metrics" key?

Reply:
As discussed offline: due to restrictions of existing APIs (protobufs and cirq's JSON serialization protocol) making serialization format more human readable requires jumping through some hoops that complicates the logic. Code readability is more important than serialization format readability, so feel free to keep as is.